src/riak_cs_delete_fsm.erl

```diff
@@ -50,8 +50,9 @@
                 uuid :: binary(),
                 manifest :: lfs_manifest(),
                 riak_client :: riak_client(),
-                gc_worker_pid :: pid(),
-                %% Key in GC bucket which this manifest belongs to.
+                finished_callback :: fun(),
+                %% For GC, Key in GC bucket which this manifest belongs to.
```

Comment:
This is not a typo. Two sentences:

`gc_key` is ...
- For GC: Key in GC bucket ...
- For acteive deletion, not used...